src/pages/Filter/FilterPage.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./FilterPage.module.css";
+import GenreTag from "../../components/GenreTag/GenreTag";
+import Button from "../../components/Buttons/Button";
+import { useState } from "react";
+
+function SelectGenre() {
+  const [tags, setTags] = useState([
```

Comment:
I would explicit set the type: 

```suggestion
  const [tags, setTags] = useState<Tag[]>([
```

https://github.com/typescript-cheatsheets/react/blob/main/README.md#basic-cheatsheet-table-of-contents